Fix broken build

Cf. https://travis-ci.org/TestRoots/watchdog

`[ERROR] The specified file '/home/travis/build/TestRoots/watchdog/intellij/build_cache/.zip' not exists`

--- a/intellij/fetchidea.sh
+++ b/intellij/fetchidea.sh
@@ -14,7 +14,7 @@ cd $build_dir
 if [ ! -f $idea_zip ];
    then
    echo "File $idea_zip not found. Loading from the Internetz ..."
-   wget $idea_URL || { echo "Failed to download IntelliJ, you probably need to update the version tp a version listed on https://www.jetbrains.com/idea/download/previous.html"; exit 1; }
+   wget $idea_URL || { echo "Failed to download IntelliJ, you probably need to update the version to a version listed on https://www.jetbrains.com/idea/download/previous.html"; exit 1; }
 fi
 
 has_directory=$(find . -type d -name 'idea-IU*' | head -n 1)

--- a/intellij/fetchidea.sh
+++ b/intellij/fetchidea.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-idea_version="2017.2.5"
+idea_version="2017.2.6"
 idea_zip="ideaIU-$idea_version.tar.gz"
 idea_URL="https://download.jetbrains.com/idea/$idea_zip"
 script_dir=$(dirname "$(readlink -f "$0")")
@@ -14,7 +14,7 @@ cd $build_dir
 if [ ! -f $idea_zip ];
    then
    echo "File $idea_zip not found. Loading from the Internetz ..."
-   wget $idea_URL
+   wget $idea_URL || { echo "Failed to download IntelliJ, you probably need to update the version tp a version listed on https://www.jetbrains.com/idea/download/previous.html"; exit 1; }
 fi
 
 has_directory=$(find . -type d -name 'idea-IU*' | head -n 1)

--- a/intellij/pom.xml
+++ b/intellij/pom.xml
@@ -22,7 +22,7 @@
 		<dependency>
 			<groupId>org.jetbrains</groupId>
 			<artifactId>org.jetbrains.intellij-ce</artifactId>
-			<version>2017.2.5</version>
+			<version>2017.2.6</version>
 			<type>zip</type>
 			<scope>provided</scope>
 		</dependency>

